test/test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
So nice tests here, but per lab instructions you actually need to test your endpoints, instead of your model.